internal/cache/passwddb_test.go

```diff
@@ -163,20 +159,18 @@ func TestNextPasswdEntry(t *testing.T) {
 	}
 
 	cacheDir := t.TempDir()
-	startTime := time.Now()
-	insertUsersInDb(t, cacheDir)
-	endTime := time.Now()
+	testutils.PrepareDBsForTests(t, cacheDir, "users_in_db")
 
-	c := newCacheForTests(t, cacheDir, cache.WithTeardownDuration(0), cache.WithOfflineCredentialsExpiration(0))
+	c := testutils.NewCacheForTests(t, cacheDir)
 
 	// Iterate over all entries
 	numIteration := len(wanted)
 	for i := 0; i < numIteration; i++ {
 		u, err := c.NextPasswdEntry(context.Background())
 		require.NoError(t, err, "NextPasswdEntry should initiate and returns values without any error")
 
-		// LastOnlineAuth should be recent
-		assert.True(t, testutils.TimeBetweenOrEquals(u.LastOnlineAuth, startTime, endTime), "Last Online auth should match insertion time. Last Online auth: %v. Start: %v, End: %v", u.LastOnlineAuth, startTime, endTime)
+		// LastOnlineAuth should be greater than 0
+		assert.False(t, u.LastOnlineAuth.IsZero(), "Last Online should not be zero.")
```

Comment:
oh, there is no Greater/Less?

Reply:
There is, but then I would have to either compare it with the 0 value for time, which is weird, or I would have to convert the time to Unix and them compare it with 0. Since the time package already has this builtin function (which, I think, would compare the Time more efficiently), I thought it would be better to use it.

Reply:
Looks at the rest of the code to get inspiration :)

Right now, I’m comparing to the 0 time, but I do not have strong preference. However, we should only do it one way throughout the code.